### ISSUE TITLE
Critical: DBeaver uses different DB than shown

### ISSUE THREAD

**Issue by 2019-05-10** [dbeaver/dbeaver#10171]
#### System information: 
- Linux, Debian/SID
- DBeaver version: 
- Java version 
- Additional extensions

#### Connection specification:
- Database name and version Version 7.2.3.202010181902
- Driver name Postgres JDBC
- Do you use tunnels or proxies (SSH, SOCKS, etc)? No

#### Describe the problem you're observing:
I got several development databases, which are copies of each other, starting with
mc3
which is configured as the default DB of the connection
Next version is
mc3b
and the current one is
mc4b

On friday the DBeaver editor was connected to 
mc4b
when I closed DBeaver.

Today I opened DBeaver again w/o any changes 
- "Active catalog/schema" dropdown correctly shows "mc4b"
- tooltip of editor correctly shows "mc4b"
- "SELECT current_database();" _incorrectly_ shows "**mc3b**"!

Disconnection/reconnection did not change anything.
Only after I restarted DBeaver in all three places the DB name matched and the results were as expected.

I only noticed that b/c I knew for a fact that several tables had to be empty (both my application I am developing and psql confirmed that). And yet, I was close to delete that data again, convinced I was operating in the right database.

#### Steps to reproduce, if exist:

- open editor for a certain database's table
- close DBeaver
- start DBeaver again
- check with SELECT current_database();

![Screenshot_20201026_130756](https://user-images.githubusercontent.com/50481865/97172670-a4592580-178f-11eb-810c-928cd8382682.png)

**Comment by uslss** [comment]
@2019-05-10 
cannot reproduce it. Does it reproducible always for you?

**Comment by 2019-05-10** [comment]
So far, it didn't.
What I noticed: the editor in question was for a file I accidentally did not save in the workspace, but the system temp folder.
I moved the file to workspace's Scripts folder (~/.local/share/.DBeaverData/workspace6/General/Scripts/ where all other files live) without overwriting a previously file with the same name.

Don't know how the DB _name_shown_ (dropdown and tooltip) and the DB _actually_connected_to_ are related, but would it be possible that the associated metadata got confused due to the fact that the editor was opened from a non-standard location?
